tests/unit/test_list_outputs.py

```diff
@@ -0,0 +1,175 @@
+import json
+
+import pytest
+
+from cromshell.list_outputs import command as list_outputs_command
+
+
+class TestListOutputs:
+    """Test the execution list-outputs command functions"""
+
+    # def get_workflow_level_outputs(config) -> dict:
```

Comment:
is this a self-reminder? If so, let's be a bit explicit in the comments.

Reply:
oops, that should have been deleted